tests/integration/test_requirements_dev.py

```
@@ -147,12 +162,13 @@
     
     def test_version_format_valid(self, requirements: List[Tuple[str, str]]):
-        """Test that version specifications use valid format."""
-        version_pattern = re.compile(r'^(>=|==|<=|>|<|~=)\d+(\.\d+)*$')
-        
+        """Test that version specifications use valid PEP 440 format."""
         for pkg, ver_spec in requirements:
             if ver_spec:
-                assert version_pattern.match(ver_spec)
+                try:
+                    SpecifierSet(ver_spec)
+                except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)